### PR TITLE
Sarah

### PR DESCRIPTION
**Pull request by sgleix**
Added basic filtering, page navigation, search bar

### CODE REVIEW

**Review by illusion173 — APPROVED** [review]
LGTM

**Review by illusion173 — APPROVED** [review]
LGTM

File: code/websitesrc/cybertoolwebsite/amplify/backend/types/amplify-dependent-resources-ref.d.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export type AmplifyDependentResourcesAttributes = {
+<<<<<<< Updated upstream
```
Comment (TroyNeubauer) [inline code/websitesrc/cybertoolwebsite/amplify/backend/types/amplify-dependent-resources-ref.d.ts:2]: Looks like committed merge conflicts. Guessing this is not what you want?

Comment (illusion173) [reply code/websitesrc/cybertoolwebsite/amplify/backend/types/amplify-dependent-resources-ref.d.ts:2]: Anything in the amplify/* director is subject to change and expected to conflict. Everytime you have to run the website locally anyway, you have to run the appropriate amplify pull command given by the AWS website. This command can and will overwrite/create files. If anything we can put the entire amplify directory in gitignore.

**Pull request merged**